engine/apps/integrations/urls.py

```diff
@@ -33,7 +32,6 @@
     path("grafana_alerting/<str:alert_channel_key>/", GrafanaAlertingAPIView.as_view(), name="grafana_alerting"),
     path("alertmanager/<str:alert_channel_key>/", AlertManagerAPIView.as_view(), name="alertmanager"),
     path("amazon_sns/<str:alert_channel_key>/", AmazonSNS.as_view(), name="amazon_sns"),
-    path("heartbeat/<str:alert_channel_key>/", HeartBeatAPIView.as_view(), name="heartbeat"),
```

Comment:
there doesn't appear to be any traffic to this endpoint, fairly certain it is safe to remove, alongside the model/table. All new heartbeats appear to be of the `IntegrationHeartBeat` type